test/scripts/bootstrap-android-emulator.sh

```diff
@@ -32,17 +56,35 @@ if [ -z "$sdkmanager" ]; then
   sdkmanager="$ANDROID_HOME/cmdline-tools/tools/bin/sdkmanager"
 fi
 
+if [ ! -f "$avdmanager" ]; then
+  echo "not ok - Unable to locate avdmanager."
+  exit_and_write_code 1
+fi
+
+if [ ! -f "$sdkmanager" ]; then
+  echo "not ok - Unable to locate sdkmanager."
+  exit_and_write_code 1
+fi
+
+if [ ! -f "$emulator" ]; then
+  echo "not ok - Unable to locate emulator."
+  exit_and_write_code 1
+fi
+
+write_code 0
+
+emulator_flags+=(
+  -camera-back none
+  -no-boot-anim
+  -no-window
+  -noaudio
+)
+
 if ! "$avdmanager" list avd | grep 'Name: SSCAVD$'; then
-  pkg="system-images;android-33;google_apis;$(uname -m | sed -E 's/(arm64|aarch64)/arm64-v8a/g')"
-  "$sdkmanager" "$pkg" || exit $?
-  echo yes | "$avdmanager" --clear-cache create avd -n SSCAVD -k "$pkg" -d 1 --force || exit $?
-fi
-
-"$emulator" @SSCAVD         \
```

Comment:
Why drop `$EMULATOR_FLAGS` and the `-gpu swiftshader_indirect` flag?

Reply:
Bad rebase, fixed now 🙏